File: app/src/main/java/com/decode/tumblr/model/Blog.java
package com.decode.tumblr.model;

import com.google.gson.annotations.SerializedName;

public class Blog {

    @SerializedName("description")
    public String description;

    @SerializedName("title")
    public String title;

    @SerializedName("updated")
    public String updated;

    @SerializedName("url")
    public String url;

    @SerializedName("total_posts")
    public int total_posts;


}

File: app/src/main/java/com/decode/tumblr/data/model/Blog_.kt
package com.decode.tumblr.data.model

import com.google.gson.annotations.SerializedName

data class Blog_(

        @SerializedName("name")
        var name: String? = null,

        @SerializedName("active")
        var active: Boolean? = null,

        @SerializedName("theme")
        var theme: Theme? = null,

        @SerializedName("share_likes")
        var shareLikes: Boolean? = null,

        @SerializedName("share_following")
        var shareFollowing: Boolean? = null,

        @SerializedName("can_be_followed")
        var canBeFollowed: Boolean? = null

)
